gitlab3/_api_definition.py

```diff
@@ -561,15 +580,7 @@ def wrapped(*args, **kwargs):
                 return ret
             return wrapped
 
-    class SearchForUserAction(ExtraActionDefinition):
-        url = '/users?search=:q'
-        method = _HTTP_GET
-
-    extra_actions = [
-        AddProjectForUserAction,
-        FindProjectsByNameAction,
-        SearchForUserAction,
-    ]
```

Comment:
The search action vas been changed on gitlab 8.* with :

/users?username=$usernameSearch 
/users?email=$emailSearch 
/users?id=$idSearch 
.....

If you want i could modify the function